ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template1.kt

```diff
@@ -53,12 +55,10 @@ import com.revenuecat.purchases.ui.revenuecatui.extensions.introEligibility
 @Composable
 internal fun Template1(state: PaywallState.Loaded, viewModel: PaywallViewModel) {
     Column(
-        modifier = Modifier.fillMaxSize(),
+        modifier = Modifier.fillMaxWidth(),
         horizontalAlignment = Alignment.CenterHorizontally,
     ) {
-        Column(modifier = Modifier.weight(1f)) {
```

Comment:
I had to change the layout a bit to support footer.

Reply:
There are slight differences in the space between elements, since we are now relying on the spacers instead of the verticalArrangment spacebetween, but I think it looks mostly the same

Reply:
But yeah, I will try to get some snapshot tests running tomorrow 🤞